File: Beecrowd/src/main/java/Iniciante/BEE_1020.java
package Iniciante;
import java.util.Scanner;

public class BEE_1020 {
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        int Idade = in.nextInt();
        int Anos = Idade / 365;
        int Meses = (Idade - (Anos * 365)) / 30;
        int Dias = Idade - (Anos * 365) - (Meses * 30);
        
        System.out.println(Anos+" ano(s)");
        System.out.println(Meses+" mes(es)");
        System.out.println(Dias+(" dia(s)"));
    }
    
}

File: JavaScript/BEE_1020.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var N = parseInt(input)
var anos = parseInt(N / 365)
var meses = parseInt((N - (anos * 365)) / 30)
var dias = parseInt(N - (anos * 365) - (meses * 30))

console.log(`${anos} ano(s)`)
console.log(`${meses} mes(es)`)
console.log(`${dias} dia(s)`)